reference/procedural-proto-nodes.md

```diff
@@ -102,11 +102,20 @@ Using the [lua-gd](http://ittner.github.io/lua-gd) module it is possible to gene
 
 In addition to these fonts, it is possible to add other TrueType fonts file in your `PROJECT_HOME/fonts` directory.
 
+### Optimization
+
+Using procedural PROTOs can greatly increase the lodaing time of your worlds because every procedural PROTO need to be evaluated.
```

Comment:
Not many months ago, we removed all the `PROTOs` plural form and replaced them by plural name like `PROTO nodes`, `PROTO files`, `PROTO models`, etc.